templates/JazzyFile.stencil

```diff
@@ -0,0 +1,22 @@
+author: Microsoft
+author_url: https://azure.github.io/azure-sdk/
+github_url: https://github.com/Azure/azure-sdk-for-ios
+module: {{ model.name }}
+module_version: 0.1
+readme: ../sdk/{{ model.namespace }}/{{ model.name }}/README.md
+skip_undocumented: false
+hide_unlisted_documentation: false
+theme: fullwidth
+output: ../build/jazzy/{{ model.name }}
+clean: true
+sdk: iphonesimulator
+xcodebuild_arguments:
+  - "-workspace"
+  - "AzureSDK.xcworkspace"
```

Comment:
The generated code does not have a workspace generated, Will `"AzureSDK.xcworkspace"` work?

Reply:
I'm trying to test that now. Right now the model is that we generate into the azure-sdk-for-ios repo, so it should eventually land in that workspace.